Source/monster.h

```diff
@@ -48,20 +49,21 @@ enum monster_flag : uint16_t {
 };
 
 /** Indexes from UniqueMonstersData array for special unique monsters (usually quest related) */
-enum : uint8_t {
-	UMT_GARBUD,
-	UMT_SKELKING,
-	UMT_ZHAR,
-	UMT_SNOTSPIL,
-	UMT_LAZARUS,
-	UMT_RED_VEX,
-	UMT_BLACKJADE,
-	UMT_LACHDAN,
-	UMT_WARLORD,
-	UMT_BUTCHER,
-	UMT_HORKDMN,
-	UMT_DEFILER,
-	UMT_NAKRUL,
+enum class UniqueMonsterType : uint8_t {
+	Garbud,
+	SkeletonKing,
+	Zhar,
+	SnotSpill,
+	Lazarus,
+	RedVex,
+	BlackJade,
+	Lachdan,
+	WarlordOfBlood,
+	Butcher,
+	HorkDemon,
+	Defiler,
+	NaKrul,
+	None = static_cast<uint8_t>(-1),

```

Comment:
can just change the type to `int8_t`, it's not like there are more then 128 uniques :)